database/schema/upgrade_scripts/127-system_cve_data.sql

```diff
@@ -0,0 +1,73 @@
+-- system_cve_data
+CREATE TABLE IF NOT EXISTS system_cve_data (
```

Comment:
Have you considered maybe using 
```
CREATE TABLE system_cve_data AS (
   SELECT system_id, cve_id, status_id, status_text
   FROM system_vulnerabilities_active
   WHERE status_id != 0 OR status_text IS NOT NULL
);
```
Not sure if this would be better, or you want to table to be empty at the start. Just a thought

Reply:
Not sure, It'd fill the data but I think I'd need to add constraints to the table anyway.